js/listview.js

```diff
@@ -251,3 +288,16 @@ function sorted(activities) {
 	});
 	return result;
 }
+
+function scrollbar_position() {
+	var diff = scrollbarTop - scrollbarMaxTop;
+	if (diff > 0) {
+		return (scrollbarTop - diff);
+	}
+	return scrollbarTop;
+}
+
+window.addEventListener("beforeunload", (e) => {
```

Comment:
Not a good idea to use this type of global event. Will be better to save position before running activity [here](https://github.com/llaske/sugarizer/blob/dev/lib/settings.js#L413).